client/landing/stepper/declarative-flow/internals/steps-repository/plans/style.scss

```diff
@@ -0,0 +1,158 @@
+			text-align: center;
+		}
+	}
+
+	.plans__loading-container {
+		margin-top: 18em;
+	}
+
+	&.is-wide-layout {
+		margin: auto;
+		max-width: 1200px;
+	}
+
+	.formatted-header.is-without-subhead {
+		margin-bottom: 15px;
+	}
+
+	.step-wrapper {
+		&.is-wide-layout {
+			max-width: 1200px;
+		}
+	}
+
+	.plan-features-comparison__table {
+		.plan-features-comparison__actions {
+			button.plan-features__actions-button {
+				border-radius: 4px;
+			}
+		}
+
+		.plan-features-comparison__row:last-child {
+			.plan-features-comparison__table-item {
+				border-bottom: none;
+			}
+		}
+		.plan-features-comparison__table-item {
+			border-left: none;
+			background-color: transparent;
+
+			&:last-of-type {
+				border-right: none;
+			}
+
+			.plan-pill.is-in-signup {
+				font-size: 0.75rem;
+				font-weight: 500;
+				letter-spacing: 0.2px;
+				/* stylelint-disable-next-line declaration-property-unit-allowed-list */
+				line-height: 1.25rem;
+				padding: 0 8px;
+				right: 52%;
+				transform: translate(50%);
+				border-radius: 4px;
+				background-color: var(--studio-green-5);
+				color: var(--studio-gray-80);
+			}
+			.plan-features-comparison__item-annual-plan {
+				color: var(--studio-orange-40) !important;
+			}
+		}
+	}
+
+	button.is-borderless {
+		color: var(--studio-gray-90);
+		font-weight: 500;
+		border: 0;
+		box-shadow: none;
+		font-size: inherit;
+		line-height: inherit;
+		padding: 0;
+		text-decoration: underline;
+	}
+
+	.plans-features-main {
+		ul.segmented-control {
+			background-color: #f2f2f2;
+			border-radius: 6px;  /* stylelint-disable-line scales/radii */
+			border-color: var(--studio-gray-5);
+
+			li.segmented-control__item {
+				border: 6px;
+				padding: 2px;
+				&:first-child {
+					margin-right: 5px;
+				}
+				&.is-selected {
+					a.segmented-control__link {
+						padding-top: 6px;
+						padding-bottom: 6px;
+						border: 0.5px solid #0000000a;
+						background-color: var(--studio-white);
+						color: var(--color-text);
+						border-radius: 5px;  /* stylelint-disable-line scales/radii */
+						box-shadow: 0 3px 8px #0000001f, 0 3px 1px #0000000a;
+					}
+				}
+				a.segmented-control__link {
+					border: solid 1px #f2f2f2;
+					border-radius: 5px;  /* stylelint-disable-line scales/radii */
+				}
+			}
+			li.segmented-control__item:not(.is-selected) {
+				a.segmented-control__link {
+					border: solid 1px #f2f2f2;
+					border-radius: 5px;  /* stylelint-disable-line scales/radii */
+					&:hover {
+						background-color: unset !important;
+						border: 1px solid var(--studio-gray-10);
+						border-radius: 5px;  /* stylelint-disable-line scales/radii */
+					}
+				}
+			}
+		}
+	}
+}
+
+.plans-step__compare-plans-link {
+	clear: both;
+	display: block;
+	font-size: $font-body-extra-small;
+	margin: 20px 0 0;
+	text-align: center;
+
+	.gridicon {
+		margin: -2px 3px 0 0;
+		vertical-align: middle;
+	}
+}
+
+.plans-step__compare {
+	margin-top: 25px;
+}
```

Comment:
Is this doing anything?

Reply:
nope, removed. It was from 'old code'
thank you